schemas/pointer.schema.yaml

```
@@ -11,9 +11,13 @@ description: |
 
 type: object
 
-oneOf:
-  - $ref: "schema:ethdebug/format/pointer/region"
-  - $ref: "schema:ethdebug/format/pointer/collection"
+if:
+  properties:
+    location: {}
```

This is probably failing because here's the right way to do this:

```suggestion
    required: [location]
```

This is a bit non-intuitive, but think of `if` as a stand-alone schema. A schema that just defines `location` as an optional property matches every object, whether or not it has `location`. We want to check only for schemas that definitely have `location`, so our `if` schema must check specifically for it being required.

Let me know if this doesn't make sense and I can try explaining it a different way.